package.json

```diff
@@ -38,7 +38,6 @@
     "remark": "^14.0.3",
     "remark-gfm": "^3.0.1",
     "remark-prism": "^1.3.6",
```

Comment:
The updates to dependencies aren't here, only the removal of `request-ip`

Reply:
OMG really 😱 thank you!

Reply:
The package lock file has been updated with the latest changes https://github.com/EddieHubCommunity/LinkFree/pull/8337/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519

Reply:
oh yes that is confusing! I will do that now, thank you 👍